projects/dev.yaml

```diff
@@ -25,6 +25,10 @@ paths:
                 - path: v1/mobileapps.yaml
                   options: '{{merge({"response_cache_control": options.purged_cache_control},
                                 options.mobileapps)}}'
+                - path: v1/media.yaml
+                  options:
+                    response_cache_control: '{{options.purged_cache_control}}'
```

Comment:
If this is purged we need a ChangeProp rule to purge it. I'll make one.

Reply:
Yes, the other Page Content Service endpoints (except the base CSS one) will need the same.

Reply:
I guess it's fine to assume then whenever Parsoid HTML changes -> media might also change so we need a rule for parsoid HTML change?